<inh f='include/c++/4.8.2/exception' l='68' c='_ZNKSt9exception4whatEv'/>
<def f='vpp_1804/src/vpp-api/vapi/vapi.hpp' l='70' ll='73' type='const char * vapi::Msg_not_available_exception::what() const'/>
